docs/config/README.md

```diff
@@ -57,6 +58,28 @@ This is the list of prompt-wide configuration options.
 
 # Disable the newline at the start of the prompt
 add_newline = false
+prompt_order=["rust","line_break","package","line_break","character"]
+```
+
+### default_prompt_order
```

Comment:
I'm not 100% sure about exposing this in the docs right now: we currently have two other PRs in the pipeline to add new modules and several additional issues about modules beyond that. If we throw this in now, we'll have to make sure that this and `print.rs` stay in sync.

I do think it's a good idea, but we might want to save it for 1.0.

Reply:
I have removed expose of default_prompt_order since I also believe it is not good idea to add this currently with other PR open. Can implement this in near future after addition of all language module

Reply:
But after merging of this PR other PR open needs to add out added module in `ALL_MODULE` const in `print.rs` to support out custom prompt order for those module but need not to update config README